corehq/apps/reports/standard/cases/basic.py

```diff
@@ -24,6 +26,129 @@
 from .data_sources import CaseInfo, CaseDisplay
 
 
+class CaseExportScope(object):
```

Comment:
This does change in logic and is not an exact over write. This extract is also used later for any place that needs to add this scope for example in Case Reassignment. There are a lot of classes that inherit from CaseListMixin. I am not sure about adding changes to all those classes by updating CaseListMixin, hence extracted this out. This extract can be used for any places we make locations restricted.